app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -158,36 +143,32 @@ private fun EditProfileContent(
   InputText(
       value = firstName,
       onValueChange = onFirstNameChange,
-      label = stringResource(R.string.view_profile_screen_first_name_label),
+      label = "First Name",
```

Comment:
Please try and keep constants strings in the `.xml` files like @andrepeiry did before on this screen

Reply:
Noted